sellotape/sellotape_dj/chat/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+# Create your models here.
+class Message(models.Model):
+    author = models.ForeignKey(User, related_name = 'author_messages', on_delete = models.CASCADE)
```

Comment:
We have extended the regular Django `User` attributes, and we have called it a `Profile`, it should correspond with the `author` attribute on your `models.py` - chat app.
Plus, I think the `Message` object should be on the README.MD as you are creating a new ORM.
See PR #27 , it supposed to clarify the objects modeling.

Reply:
Do note that if you change this, you'd also need a new migration.

Reply:
@tarelmadar Did you address this?